src/main/java/org/mule/extension/db/internal/domain/connection/oracle/OracleConnectionUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.db.internal.domain.connection.oracle;
+
+/**
+ * Utils for the oracle db connection
+ *
+ */
+public class OracleConnectionUtils {
+
+  public static String getOwnerFrom(String typeName) {
+    if (!typeName.contains(".")) {
+      return null;
+    } else {
+      return typeName.substring(0, typeName.indexOf("."));
+    }
+  }
+
+  public static String getTypeSimpleName(String typeName) {
+    if (!typeName.contains(".")) {
+      return typeName;
+    } else {
+      return typeName.substring(typeName.indexOf(".") + 1);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Put single-quotes around '.' to use the faster "indexOf(char)" method. ([squid:S3027](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3027))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-db-connector&pullRequest=156&issues=AWkWM61JotIXnmsUtJ_Y&open=AWkWM61JotIXnmsUtJ_Y)

[ISSUE_KEY]: AWkWM61JotIXnmsUtJ_Y